### PR TITLE
add template hook in user div

### PR DESCRIPTION
**Pull request by tsiegleauq**
Adds a template hook in the user menu
[openslides-conversations](https://github.com/OpenSlides/openslides-conversations) uses that hook for adding global [microphone and camera indicators](https://github.com/OpenSlides/openslides-conversations/blob/master/openslides_conversations/static/templates/openslides_conversations/userbar-hook.html)

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
@tsiegleauq Please rebase on top of master (to fix travis).
@FinnStutzenstein Review please.

File: openslides/core/static/templates/index.html [hunk 1]
```diff
@@ -39,7 +39,9 @@
     <div class="user" ng-controller="userMenu">
       <div class="btn-group">
         <div ng-if="operator.isAuthenticated()">
-
```
Comment (FinnStutzenstein) [inline openslides/core/static/templates/index.html]: Why does git recognize this line in fact i doesn't changed. Or does it? (please check your line endings)

**Comment by tsiegleauq** [comment]
OK. Rebased it and removed the empty line.

**Comment by normanjaeckel** [comment]
Ok.

**Pull request merged**